tensorflow/contrib/verbs/rdma.cc

```diff
@@ -1064,6 +1068,7 @@ Rendezvous::DoneCallback RdmaTensorBuffer::getRecvTensorCallback(
                                  &proto, NULL, send_args, recv_args);
             });
       }
+#endif  // GOOGLE_CUDA
```

Comment:
like in [the GRPC implementation](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/distributed_runtime/rpc/grpc_worker_service.cc#L369) , maybe we can add 

#else
              done(errors::Internal("No GPU device in process"));

Reply:
We decided not to add it, since it requires changes in the function signature (need to pass the callback done).